core/src/utils/test/playwright/docs/api.md

```diff
@@ -0,0 +1,458 @@
+# Playwright Test Utils
+
+The testing directory within Ionic's codebase contains utilities that can be used to more easily test Stencil projects with Playwright.
+
+## Table of Contents
+
+- [`test` function](#test-function)
+- [`page` fixture](#page-fixture)
+- [Generators](#generators)
+- [Matchers](#matchers)
+
+## `test` Function
+
+The default [`test` function](https://playwright.dev/docs/api/class-test) has been extended to provide two custom options.
+
+| Fixture | Type | Description |
+| ------- | ---- | ----------- |
+| page    | [E2EPage](https://github.com/ionic-team/ionic-framework/blob/main/core/src/utils/test/playwright/playwright-declarations.ts) | An extension of the base `page` test fixture within Playwright |
+| skip    | [E2ESkip](https://github.com/ionic-team/ionic-framework/blob/main/core/src/utils/test/playwright/playwright-declarations.ts) | Used to skip tests based on text direction, mode, or browser |
+
+<details>
+  
+<summary>Usage</summary>
+
+**`page`**
+
+```typescript
+import { configs, test } from '@utils/test/playwright';
+
+configs().forEach(({ config, title }) => {
+  test.describe(title('my test block'), () => {
+    test('my custom test', ({ page }) => {
+      await page.goto('path/to/file', config);
+    });
+  });
+});
+```
+
```

Comment:
For consistency, add the following: `Deprecated: Use a [generator](#generators) instead.` This would let each section of DEPRECATED to have this.

Or I recommend to just add a note that any deprecation should use generators instead.

Reply:
Each section has a note below it to use a generator:

> Deprecated: Use a [generator](#generators) instead.


However, I'll take this as a sign that the note is hard to find and will make this more discoverable.

Reply:
I moved the deprecation note to be the first line: https://github.com/ionic-team/ionic-framework/pull/27248/commits/3d587bf5e672d61f673d5a5c202e0939c2afd6b5